Modules/Filtering/ImageStatistics/include/itkLabelOverlapMeasuresImageFilter.hxx

```diff
@@ -375,15 +375,20 @@ LabelOverlapMeasuresImageFilter<TLabelImage>::GetFalsePositiveError() const -> R
 {
   RealType numerator = 0.0;
   RealType denominator = 0.0;
+
+  auto sourceImg = const_cast<LabelImageType *>(this->GetInput(0));
```

Comment:
Thanks for fixing this part. So, it was "GetInput(0)".  I am still not familiar with the most parts of ITK, and I have not installed ITK c++ on my local machine. So, I relied on the PR test in there and it might cause too much update in short time.. , which is not desirable.

Should there be Python Wrapper as well? If I want to see this update in SITK in the end?

Reply:
This class is already wrapped for Python. The new methods should be automatically exposed.